docker/Dockerfile

```diff
@@ -4,6 +4,10 @@ FROM socrata/java
 ENV SERVER_NAME region-coder
 ENV SERVICE_PORT 2021
 EXPOSE 2021
+ENV JMX_PORT 9999
+ENV JMX_RMI_PORT 9998
```

Comment:
BTW I tried the one-port thing and it didn't work for me.

On Mon, May 11, 2015 at 4:49 PM, Marc Slemko <notifications@github.com>
wrote:

> In docker/Dockerfile
> <https://github.com/socrata-platform/region-coder/pull/2#discussion_r30093724>
> :
>
> > @@ -4,6 +4,10 @@ FROM socrata/java
> >  ENV SERVER_NAME region-coder
> >  ENV SERVICE_PORT 2021
> >  EXPOSE 2021
> > +ENV JMX_PORT 9999
> > +ENV JMX_RMI_PORT 9998
>
> So is the idea every process will have their own separate ports mapped to
> the same port on the host? If so we should try to (1) see if we can use
> only one port as per
> http://stackoverflow.com/questions/20884353/why-java-opens-3-ports-when-jmx-is-configured
> and (2) put the port in the rage that the app port is for profit.
>
> If not, we need to tell the port mapping to JMX/RMI somehow?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/socrata-platform/region-coder/pull/2/files#r30093724>.
>



-- 
Evan Chan   |   Principal Engineer, Socrata     |    @evanfchan
*Socrata <http://www.socrata.com/> *| The Data Experience Company™
*Socrata isn't about worldly success, but about changing the world
successfully. This is why I'm excited to be here.*
*What does Socrata do? <https://www.youtube.com/watch?v=yyz86MlkriQ> *(watch
our 2 min video)